OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/createDomainAndStart.sh

```diff
@@ -64,11 +65,11 @@ trap _int SIGINT
 trap _term SIGTERM
 
 # Set SIGKILL handler
-trap _kill SIGKILL
+# Commented out to avoid Lint shell script check failure issue during git push  
+# trap _kill SIGKILL
```

Comment:
The idea was just to try and see if shell script check passes by commenting out.  The check passed this step, however it is failing for even warnings. Removal of  echo statement for DB_PASSWORD was the only intended change. rest all are suggested by linter. Just wanted to understand if this linter feature is newly introduced, if so we may have to close the PR and rework on this script for all suggested changes.

Reply:
The linter was introduced to improve the quality and security of the scripts and documentation in this repo. It's part of a broader organisational shift to doing more automated content checking in general.

Note that the linter is running [`Markdownlint`](https://github.com/DavidAnson/markdownlint), [`shellcheck`](https://www.shellcheck.net/) and [`shfmt`](https://github.com/mvdan/sh), all of which are also available as VSCode plugins. You can (and should) run the linters locally before you push so that you know the submission will pass.

Lastly, you can also tell the linter to ignore certain checks but _use